### DISCUSSION TITLE
Welcome to Subtitle Edit Discussions!

### DISCUSSION THREAD

**Discussion by niksedk** [SubtitleEdit/subtitleedit#4879] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OmrSi** [comment]
Glad this is finally open. Hope we have good discussions here. :)

**Reply by LeonCheung** [reply]
Leon Cheung, ♂，Freelance, love open source stuff like SE so much. :) Crazy ideas always rush in my mind, glad to help in this community, u ready? lol
